src/vs/workbench/contrib/terminal/browser/links/terminalLinkManager.ts

```diff
@@ -96,12 +100,22 @@ export class TerminalLinkManager extends DisposableStore {
 		}
 
 		// Word links
-		const wordProvider = this._instantiationService.createInstance(TerminalWordLinkProvider, this._xtermTerminal, this._capabilities, this._wrapLinkHandler.bind(this), this._tooltipCallback.bind(this));
-		this._standardLinkProviders.set(TerminalWordLinkProvider.id, wordProvider);
+		const wordDetector = this._instantiationService.createInstance(TerminalWorkLinkDetector, this._xterm);
+		const wordProvider = this._instantiationService.createInstance(TerminalLinkProviderAdapter, wordDetector);
+		wordProvider.onDidActivateLink(e => this._openLink(e));
+		this._standardLinkProviders.set(TerminalWorkLinkDetector.id, wordProvider);
+
+		// Setup link openers
```

Comment:
maybe a comment like "word links may not have an exact match, so might result in a workspace search."

Reply:
That's captured in `TerminalLinkType.Search` so probably best not to repeat